lib/minitest/mock.rb

```diff
@@ -198,6 +199,10 @@ class Object
   def stub name, val_or_callable, *block_args
     new_name = "__minitest_stub__#{name}"
 
+    if respond_to? new_name
+      raise MultipleStubError, "method :%s was already stubbed" % [name.to_s]
```

Comment:
No at all. I wanted to stay as close as possible to the current coding style in many ways and saw it done this way up the page. I assumed it was for speed since this is (or used to be faster), although speed during the raising of an error is probably not key.

I didn't give it much thought, but I am happy to use interpolation instead.